frontend/src/views/NavigationDefault.vue

```diff
@@ -12,17 +12,17 @@
     v-else app
     fixed grow>
     <v-btn :to="{name: 'home'}">
-      <span>Home</span>
+      <span>{{ $tc('components.navigation.home') }}</span>
```

Comment:
Key should be `views.navigationDefault.home` (also remember to change it in the JSON)